### ISSUE TITLE
It doesn't work on Python 3 because of raw_input

### ISSUE THREAD

**Issue by ghost** [Autodesk/quack#5]
I try to run it with python 3. But it doesn't work because of `raw_input`. 

Replaced it with `input` it works OK. 

The unicode and encode destroy the print message. I am not familiar with that, but comments out the code below make it works well both on python 2 and 3.

```
print('\033[1A' + '\033[32m' +
       str(u'\u2713'.encode('utf-8')) + '\033[37m')
```

**Comment by zonito** [comment]
Okay, I will change it... Thanks for letting us know!

**Comment by zonito** [comment]
Fixed! in latest version [quack=0.1.1](https://pypi.python.org/pypi/quack/)
